tokio/tests/time_sleep.rs

```diff
@@ -259,8 +259,7 @@ async fn reset_after_firing() {
         .poll(&mut Context::from_waker(noop_waker_ref())));
 }
 
-const NUM_LEVELS: usize = 6;
-const MAX_DURATION: u64 = (1 << (6 * NUM_LEVELS)) - 1;
+const MAX_DURATION: u64 = u64::MAX;
```

Comment:
What happens if the `Duration` is even larger? The maximum `Duration` is `u64::MAX` seconds and 999999999 nanoseconds.

Reply:
You are completely right. Let me me update the PR to actually use the real max duration.